packages/next-sample/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "@silverhand/react",
+  "rules": {
+    "consistent-default-export-name/default-export-match-filename": "off",
+    "react/react-in-jsx-scope": "off"
```

Comment:
We can remove this line if we upgrade configs later

Reply:
unfortunately is still needed, can we look into this later?

Reply:
yes